File: sqlUpdate.py
import MySQLdb
#Open database connection
db =  MySQLdb.connect (host="localhost",user="User",passwd="password",db="school")
#prep a cursor object using cursor() method; cursor is like a navigation method
db.autocommit(True)
cursor = db.cursor(MySQLdb.cursors.DictCursor)

name='Kainoa'
age=15
gradeLevel=10
#student info
sql = "UPDATE students SET age=16 WHERE name='Kainoa'"
cursor.execute(sql)
#update the age of Kainoa to 16 in the students table

cursor.close()
db.close()

File: sqlUpdate.php
<?php
$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
//creating connection and putting in login credentials
// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

$sql = "UPDATE students SET age=18 WHERE name='Bertrum'";
// Updating the age of Bertrum in the students table with UPDATE and WHERE commands
if ($conn->query($sql) === TRUE) {
    // putting the query of sql w/info into $conn as a query file
    echo "New record updated successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
#connection close
?> 
